worlds/animal-well/regions.py

```diff
@@ -767,7 +768,7 @@ class AWData(NamedTuple):
             AWData(AWType.location),
         lname.egg_jade:  # do the puzzle
             AWData(AWType.location),
-        bird_capybara_waterfall:  # take the very long pipe to the sweet egg room
+        bird_capybara_waterfall:  # fish pipe to the sweet egg room -- note: never relevant. bubble gets you here from sphere 1
             AWData(AWType.region, [[iname.bubble]]),
```

Comment:
How does bubble alone get you here from sphere 1?

Reply:
Nope, I was wrong. I didn't realize we had it listed as bubble_short for that jump. That note can be deleted.